From b3258f95778335290e7767fccb84dc9f307281ba Mon Sep 17 00:00:00 2001
From: Randolph <randolph@andestech.com>
Date: Tue, 7 May 2024 11:29:14 +0800
Subject: [PATCH 31/34] configs: fix miss spl_board_init in normal boot

When Qilai using normal boot, the L2 Cache will not enable the
instruction prefetch depth and data prefetch depth.
The normal defconfig is missing CONFIG_SPL_BOARD_INIT, the L2 cache
will not enter the probe function.

Signed-off-by: Randolph <randolph@andestech.com>
---
 configs/qilai_invecas_spl_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/qilai_invecas_spl_defconfig b/configs/qilai_invecas_spl_defconfig
index 83fb2e2cc2..4a64b93a89 100644
--- a/configs/qilai_invecas_spl_defconfig
+++ b/configs/qilai_invecas_spl_defconfig
@@ -17,6 +17,7 @@ CONFIG_SPL_LOAD_FIT_ADDRESS=0x400200000
 CONFIG_SPL_OPENSBI_LOAD_ADDR=0x400000000
 CONFIG_BOOTDELAY=3
 CONFIG_MISC_INIT_R=y
+CONFIG_SPL_BOARD_INIT=y
 CONFIG_SPL_CACHE=y
 CONFIG_SYS_PROMPT="RISC-V # "
 CONFIG_CMD_IMLS=y
-- 
2.34.1

